cmd/kubeadm/.import-restrictions

```diff
@@ -1,48 +1,20 @@
-{
-	"Rules": [
-		{
-			"SelectorRegexp": "k8s[.]io/(api/|apimachinery/|apiextensions-apiserver/|apiserver/)",
-			"AllowedPrefixes": [
-				""
-			]
-		},
-		{
-			"SelectorRegexp": "k8s[.]io/client-go/",
-			"AllowedPrefixes": [
-				""
-			]
-		},
-		{
-			"SelectorRegexp": "k8s[.]io/kubelet/",
-			"AllowedPrefixes": [
-				"k8s.io/kubelet/config/v1beta1"
-			]
-		},
-		{
-			"SelectorRegexp": "k8s[.]io/kube-openapi/",
-			"AllowedPrefixes": [
-				"k8s.io/kube-openapi/pkg/util/proto"
-			]
-		},
-		{
-			"SelectorRegexp": "k8s[.]io/kube-proxy/",
-			"AllowedPrefixes": [
-				"k8s.io/kube-proxy/config/v1alpha1"
-			]
-		},
-		{
-			"SelectorRegexp": "k8s[.]io/kubernetes",
-			"AllowedPrefixes": [
-				"k8s.io/kubernetes/cmd/kubeadm"
-			]
-		},
-		{
-			"SelectorRegexp": "gopkg[.]in",
-			"AllowedPrefixes": [
-				"gopkg.in/inf.v0",
-				"gopkg.in/square/go-jose.v2",
-				"gopkg.in/yaml.v2"
-			]
-		}
-	]
-}
+rules:
+  - selectorRegexp: k8s[.]io/kubelet/
+    allowedPrefixes:
+      - k8s.io/kubelet/config/v1beta1
+  - selectorRegexp: k8s[.]io/kube-openapi/
+    allowedPrefixes:
+      - k8s.io/kube-openapi/pkg/util/proto
+  - selectorRegexp: k8s[.]io/kube-proxy/
+    allowedPrefixes:
+      - k8s.io/kube-proxy/config/v1alpha1
+  - selectorRegexp: k8s[.]io/kubernetes
+    allowedPrefixes:
+      - k8s.io/kubernetes/cmd/kubeadm
+      # TODO: cmd/kubeadm/app/phases/controlplane has tests that use this
+      - k8s.io/kubernetes/pkg/features
```

Comment:
hm, i though we removed this dependency.
wondering where this is coming from.

i've tried with:
```
cd ./cmd/kubeadm
grep "k8s.io/kubernetes/pkg" * -rnI
go list -f '{{ .ImportPath }}' ./... | grep "k8s.io/kubernetes/pkg"
go list -f '{{ .Deps }}' ./... | grep "k8s.io/kubernetes/pkg"
go list -f '{{ .Imports }}' ./... | grep "k8s.io/kubernetes/pkg"
```

also at some point i moved kubeadm to a separate repository for a test and it did not have `k8s.io/kubernetes` in the `go.mod/sum`

Reply:
Thanks for the sanity check, looks like this was addressed by #90078